quirks/password-rules.json

```diff
@@ -626,6 +626,9 @@
     "keldoc.com": {
         "password-rules": "minlength: 12; required: lower; required: upper; required: digit; required: [!@#$%^&*];"
     },
+    "kennedy-center.org": {
+        "password-rules": "minlength: 8; required: lower; required: upper; required: digit; required: [!#$%&*?@];"
```

Comment:
Killer commit message, @vcsjones! Good enough that it makes it really easy to give you the only feedback I have: this rule should have an "allowed: lower;" at the end.

Reply:
@rmondello a lower is also required (above where I said "The site also requires a lower-case letter, I think it is just implied.").

Should the rule have both a `required: lower` _and_ an `allowed: lower` at the end?

Reply:
Wow, I failed to read this properly! This is absolutely perfect as-is!